conf/config-default.yaml

```diff
@@ -133,14 +133,13 @@ apisix:
   ssl:
     enable: true
     listen:                       # APISIX listening port in https.
-      - 9443
+      - port: 9443
+        enable_http2: true
     #   - port: 9444
```

Comment:
We can remove this comment now as it is enabled

Reply:
remove it